src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,33 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final String whiteSpace = " ";
+    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("d.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate dateFromLocal = LocalDate.parse(dateFrom, formatter);
+        LocalDate dateToLocal = LocalDate.parse(dateTo, formatter);
+        int[] salaryAll = new int[names.length];
```

Comment:
you dont need an additional array to store salaries, just add a salary info for each person inside the loop